onnxruntime/test/providers/cpu/model_tests.cc

```diff
@@ -66,6 +66,8 @@ TEST_P(ModelTest, Run) {
   if (model_info->GetONNXOpSetVersion() != 8 && provider_name == "tensorrt") {
     // TensorRT can run most of the model tests, but only part of
     // them is enabled here to save CI build time.
+    // Besides saving CI build time, TRT isn’t able to support full ONNX ops spec and therefore some testcases will fail.
```

Comment:
How about we always put the focus on the latest op set? I think TRT users don't quite care about opset 8. @jywu-msft , what do you think?

Reply:
seems like a good suggestion. opset8 was chosen a long time ago (i think it was the latest at the time)

Reply:
From official website https://github.com/onnx/onnx/blob/master/docs/Versioning.md
I saw the latest op set is 15, but the latest op set in our test models is 13.

So I will test only on opset 13 for TRT EP. One thing to notice is that there are around 68 testcases failed on testing opset 13.

Reply:
TensorRT 8.0 is supposed to have some opset 13 support, but 68 test case failures is quite a lot.
just curious how some of the other opsets look, like 11 for example.

Reply:
After we have disabled some testcases, opset 13 still has 68 failed testcases, opset 12 has 74 failed testcases and opset 11 has 12 failed testcases. 

I'm testing on CI VM with T4.

Reply:
I just added 14/15 there.

Reply:
How can I update model folder to include opset14&15 models on the VM which uses old CI image?

Reply:
Update "broken_tests" for the newly added opsets.